kvmagent/ansible/kvm.py

```
@@ -657,6 +657,15 @@ def copy_zs_scripts():
     _dst = '/usr/local/bin/'
     copy_to_remote(_src, _dst, "mode=755", host_post_info)
 
+@with_arch(todo_list=['x86_64'], host_arch=host_info.host_arch)
+def copy_tensor_fusion_worker():
+    """copy tensor-fusion-worker binary from mn_node to host_node"""
+    _src = '/opt/zstack-dvd/{}/{}/scripts/tensor-fusion-worker'.format(host_info.host_arch, releasever)
+    if os.path.exists(_src):
+        _dst = '/usr/local/bin/tensor-fusion-worker'
+        copy_to_remote(_src, _dst, "mode=755", host_post_info)
```

Ruff (0.15.5)

[error] `copy_to_remote` may be undefined, or defined from star imports

(F405)